src/BenchmarkDotNet/Portability/Cpu/HardwareIntrinsics.cs

```diff
@@ -0,0 +1,272 @@
+﻿using System.Collections.Generic;
+using BenchmarkDotNet.Environments;
+#if NET6_0_OR_GREATER
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics.Arm;
+using System.Numerics;
+#elif NETSTANDARD2_0_OR_GREATER
+using System;
+#endif
+
+namespace BenchmarkDotNet.Portability.Cpu
+{
+    internal static class HardwareIntrinsics
+    {
+        internal static string GetVectorSize()
+        {
+#if NET6_0_OR_GREATER
+            if (Vector.IsHardwareAccelerated)
+                return $"VectorSize={Vector<byte>.Count * 8}";
+#endif
```

Comment:
Why is this .Net 6+? Those methods also existed on .Net Framework and such.

Reply:
@MichalPetryka thanks, I was not aware of that!

Reply:
Just do note that there you have to install a nuget package for it.